tools/beman-tidy/beman_tidy/lib/checks/beman_standard/directory.py

```diff
@@ -19,7 +20,55 @@ def __init__(self, repo_info, beman_standard_check_config, prefix_path):
         )
 
 
-# TODO DIRECTORY.INTERFACE_HEADERS
+@register_beman_standard_check("DIRECTORY.INTERFACE_HEADERS")
+class DirectoryInterfaceHeadersCheck(DirectoryBaseCheck):
```

Comment:
```suggestion
class DirectoryInterfaceHeadersCheck(BemanTreeDirectoryCheck):
```

Should use this helper class - check https://github.com/bemanproject/infra/pull/114

Reply:
I wanted to use this helper class but it sets working directory to this:
`f"{prefix_path}/beman/{repo_info['name']}",`
but I need to do the check in the whole repo, not just in exemplar/include/beman/examplar to use as an example.